content/en/developers/authorization/oauth2_in_datadog.md

```diff
@@ -54,13 +56,14 @@ For more information about client creation and publishing, see [OAuth for Datado
 
 ### Initiate authorization from a third-party location 
 
-You can start the authorization process in Datadog by clicking **Connect Accounts** in the integration tile, or from the integration's external website. For example, if there's an integration configuration page on your website that Datadog users need to use, you can give users the option to start the authorization process from there.
 
-When kicking off authorization from a third-party location—anywhere outside of the Datadog integration tile—you need to account for the [Datadog site][17] (such as EU, US1, US3, or US5) when routing users through the authorization flow and building out the URL for the `authorization` and `token` endpoints. 
+Users can start the authorization process in Datadog by clicking **Connect Accounts** in the integration tile. When a user clicks Connect Accounts in Datadog, information regarding their [Datadog site][17] is sent on redirect to the `onboarding_url`. The user's Datadog site is required to make API calls on behalf of the user and receive an authorization code. If a user initiates authorization from the _integration's external website_, the user's site information is not provided. 
```

Comment:
We may want to use the word `domain` instead of `site` everywhere now to avoid confusion with the other url param

Reply:
I feel like officially it's called the Datadog site, while the domain param is specific to this OAuth implementation